servers.json

```diff
@@ -759,7 +754,9 @@
         "id": "pvpbar",
         "name": "PvPBar",
         "addresses": [
-            "pvp.bar"
+            "pvp.bar",
+            "eu.pvp.bar",
```

Comment:
Please remove this line, Lunar Client services automatically check for sub-domains, this is not needed.

Take a look at [this](https://github.com/LunarClient/ServerMappings#ip-addresses) section in the readme for more information on addresses.